### ISSUE TITLE
can not sign up

### ISSUE THREAD

**Issue by HoHodatudou** [warpdotdev/Warp#4612]
### Describe the bug

Whether you log in using email or login using github, the page will prompt you "Oops! We were unable to sign you in. Please try again."

### To reproduce

Whether you log in using email or login using github, the page will prompt you "Oops! We were unable to sign you in. Please try again."

### Screenshots

![error](https://github.com/warpdotdev/Warp/assets/42955926/80baf426-ef54-4d1f-a66e-dcca9c2e10ba)

### Operating system

MacOS

### Operating system and version

lastest

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

No, this same issue happens in Warp and other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by bnavetta** [comment]
Sorry about that, thanks for letting us know! Do you have another email address you could try, maybe with a different provider? Also, is that email the one that your GitHub account uses too?

**Comment by 6643** [comment]
@bnavetta 

I have the same problem.
Is the email used by the GitHub account

**Comment by 6643** [comment]
Google domain name cannot be accessed in China

**Comment by 6643** [comment]
@HoHodatudou  
大兄弟上科技
可以用

**Comment by bnavetta** [comment]
Ah, yes - because Warp uses Firebase, it needs access to [`*.googleapis.com`](https://docs.warp.dev/help/troubleshooting-login-issues#nothing-happens-when-i-click-sign-up) in order to sign up.

**Comment by HoHodatudou** [comment]
> Ah, yes - because Warp uses Firebase, it needs access to [`*.googleapis.com`](https://docs.warp.dev/help/troubleshooting-login-issues#nothing-happens-when-i-click-sign-up) in order to sign up.

Maybe it is the question. I can not reach google now.

**Comment by HoHodatudou** [comment]
> Sorry about that, thanks for letting us know! Do you have another email address you could try, maybe with a different provider? Also, is that email the one that your GitHub account uses too?

Another email address 、a different provider has the same problem.Maybe because I can not reach google?

**Comment by HoHodatudou** [comment]
> @HoHodatudou 大兄弟上科技 可以用

啥意思啊老哥

**Comment by bnavetta** [comment]
> Maybe because I can not reach google?

Yeah, I believe so - if your browser can't reach the Firebase APIs at `googleapis.com`, Warp unfortunately won't be able to sign you in.

**Comment by 6643** [comment]
@HoHodatudou 
翻墙
